### PR TITLE
Update delete-a-user.md

### LINKED ISSUE #1585: Clarify "inactivate"
In the following text:

"If you set up email forwarding, you cannot remove the license. Doing so will stop the email forwarding and inactivate the mailbox."

Can you please clarify what is the meaning of "inactivate" and/or rephrase it. If you are referring to the "inactive mailbox" functionality, make sure to explicitly mention that the mailbox needs to be on hold first. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f6245943-00b8-c47b-f739-731531d5d357
* Version Independent ID: a9ec7066-ed8c-7b77-1d87-8c50bf74ec6c
* Content: [Delete a user from your organization - Microsoft 365 admin](https://docs.microsoft.com/en-us/microsoft-365/admin/add-users/delete-a-user?view=o365-worldwide)
* Content Source: [microsoft-365/admin/add-users/delete-a-user.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/add-users/delete-a-user.md)
* Service: **o365-administration**
* GitHub Login: @cmcatee-MSFT
* Microsoft Alias: **cmcatee**

### PR DESCRIPTION
**Pull request by UpNorthCoder**
Fixes #1585

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/add-users/delete-a-user.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Since the guided experience walks through the steps to delete a user, here's how
 ## User management admin: Delete one or more users from Office 365
 
 
- **IMPORTANT**: Don't delete a user's account if you've [converted it to a shared mailbox](../email/convert-user-mailbox-to-shared-mailbox.md) or if you've set up email forwarding on the account. Those functions need the account there. If you've converted it to a shared mailbox, you can [Stop paying for the license](#stop-paying-for-the-license) from it so you aren't paying for it. If you set up email forwarding, you cannot remove the license. Doing so will stop the email forwarding and inactivate the mailbox. 
+**IMPORTANT**: Don't delete a user's account if you've [converted it to a shared mailbox](../email/convert-user-mailbox-to-shared-mailbox.md) or if you've set up email forwarding on the account. Those functions need the account to remain. A shared mailbox does not require a user license.If you've converted the account to a shared mailbox you can [Stop paying for the license](#stop-paying-for-the-license).If you have set up email forwarding on the account, you cannot remove the license. Doing so will stop email forwarding and deactivate the mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/delete-a-user.md]:
```suggestion
> [!IMPORTANT]
> Don't delete a user's account if you've [converted it to a shared mailbox](../email/convert-user-mailbox-to-shared-mailbox.md) or if you've set up email forwarding on the account. Those functions need the account to remain. A shared mailbox does not require a user license. If you've converted the account to a shared mailbox you can [Stop paying for the license](#stop-paying-for-the-license). If you have set up email forwarding on the account, you cannot remove the license. Doing so will stop email forwarding and deactivate the mailbox.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - suggestion. Also add spaces after periods.

**Comment by mypil** [comment]
@D3Design  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@cmcatee-MSFT - PR has been copy edited and is ready for final review. This is based on issue #1585.

Thank you.

cc: @kenwith

File: microsoft-365/admin/add-users/delete-a-user.md [hunk 2]
```diff
@@ -94,7 +94,8 @@ Since the guided experience walks through the steps to delete a user, here's how
 ## User management admin: Delete one or more users from Office 365
 
 
- **IMPORTANT**: Don't delete a user's account if you've [converted it to a shared mailbox](../email/convert-user-mailbox-to-shared-mailbox.md) or if you've set up email forwarding on the account. Those functions need the account there. If you've converted it to a shared mailbox, you can [Stop paying for the license](#stop-paying-for-the-license) from it so you aren't paying for it. If you set up email forwarding, you cannot remove the license. Doing so will stop the email forwarding and inactivate the mailbox. 
+> [!IMPORTANT]
+> Don't delete a user's account if you've [converted it to a shared mailbox](../email/convert-user-mailbox-to-shared-mailbox.md) or if you've set up email forwarding on the account. Those functions need the account to remain. A shared mailbox does not require a user license. If you've converted the account to a shared mailbox you can [Stop paying for the license](#stop-paying-for-the-license). If you have set up email forwarding on the account, you cannot remove the license. Doing so will stop email forwarding and deactivate the mailbox.
```
Comment (cmcatee-MSFT) [inline microsoft-365/admin/add-users/delete-a-user.md]: > Don't delete a user's account if you've [converted it to a shared mailbox](../email/convert-user-mailbox-to-shared-mailbox.md) or if you've set up email forwarding on the account. Those functions still need the account. A shared mailbox doesn't require a license. If you've converted the account to a shared mailbox you can [stop paying for the license](#stop-paying-for-the-license). If you've set up email forwarding on the account, you can't remove the license. Doing so will stop email forwarding and deactivate the mailbox.

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Please update with edited paragraph as indicated. Thanks.

**Pull request merged**